src/backend/index.js

```diff
@@ -74,59 +74,32 @@ process.on('uncaughtException', err => {
   shutDown();
 });
 
-/**
- * Process a string into a list of Objects, each with a feed URL
- * @param {String} lines
- */
-function processFeedUrls(lines) {
-  return (
-    lines
-      // Divide the file into separate lines
-      .split(/\r?\n/)
-      // Basic filtering to remove any ines that don't look like a feed URL
-      .filter(line => line.startsWith('http'))
-      // Convert this into an Object of the form expected by our queue
-      .map(url => ({ url }))
-  );
-}
-
 /**
  * Adds feed URL jobs to the feed queue for processing
  * @param {Array[Object]} feedJobs - list of feed URL jobs to be processed
  */
-async function enqueueFeedJobs(feedJobs) {
-  feedJobs.forEach(async feedJob => {
-    log.info(`Enqueuing Job - ${feedJob.url}`);
-    await feedQueue.add(feedJob, {
-      attempts: process.env.FEED_QUEUE_ATTEMPTS || 8,
-      backoff: {
-        type: 'exponential',
-        delay: process.env.FEED_QUEUE_DELAY_MS || 60 * 1000,
-      },
-      removeOnComplete: true,
-      removeOnFail: true,
-    });
-  });
+async function enqueueWikiFeed() {
+  const data = await wikiFeed.parseData();
+  Promise.all(
+    data.map(async feedJob => {
+      log.info(`Enqueuing feed job for ${feedJob.url}`);
+      await feedQueue.add(feedJob, {
+        attempts: 8,
```

Comment:
You are erasing changes that happened in a recent fix to make these environment variables.  Do you need to rebase?